libMultiMarkdown.h

```diff
@@ -150,6 +150,7 @@ enum keys {
 	ABBRSTART,
 	ABBRSTOP,
 	TOC,
+    ADMONITION,
```

Comment:
Looks like I need to fix some spacing here to maintain consistency.  I will do that in an update to this PR